src/tiles/grid/item-tile.ts

```diff
@@ -49,6 +57,38 @@ export class ItemTile extends LitElement {
     `;
   }
 
+  private get sortedDateInfoTemplate() {
+    let sortedValue = {} as any;
+
+    switch (this.sortParam?.field) {
```

Comment:
perhaps we use 1 function to return to return instead of the ternary in the render?

```suggestion
        if (!this.sortParam) {
          return this.creatorTemplate;
        }
    switch (this.sortParam?.field) {
```
I only note because you have `return this.creatorTemplate;` at the end of switch case

Reply:
good to have unit test for each permutation to verify what renders.

Reply:
I thought around early return but `this.sortParam?.field` can also have `titleSorter`, `titleSorter` values which is not relevant to current task and in these cases, we need to return createTemplate. that's why if sort field if not among these 4 case:, just return createTemplate as default.

Will check unit tests for these pieces.